RELEASE-NOTES.txt

```diff
@@ -1,6 +1,7 @@
 Unreleased
 ------
 * [*] Update loading and failed screens for web version of the editor [https://github.com/wordpress-mobile/gutenberg-mobile/pull/3573]
+* [*] Embed Block Implement caption field [https://github.com/wordpress-mobile/gutenberg-mobile/pull/3552]
```

Comment:
Similar to https://github.com/wordpress-mobile/gutenberg-mobile/pull/3436#discussion_r662184210, we should delay adding the embed block release notes until we unveil it from behind the dev flag.